index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
I think `String.prototype.trim` is a relatively new addition to JavaScript. [IE8 doesn't support it](http://caniuse.com/#search=trim), is that a concern?